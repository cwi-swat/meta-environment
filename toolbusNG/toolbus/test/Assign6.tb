process P is
let N : int, M : int
in
   N := 1 . 
   M := 2 . 
   printf(N, M) . 
   let N : int in N := 3 . printf(N, M) endlet . 
   printf(N, M)
endlet

toolbus(P)