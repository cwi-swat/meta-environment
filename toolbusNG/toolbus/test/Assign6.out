12
32
12
